docs/reference/migration/migrate_3_0.asciidoc

```diff
@@ -807,6 +807,17 @@ values `off`, `false`, `0`, and `no` for disabling a scripting mode.
 The variants `on`, `1`, and `yes ` for enabling and `off`, `0`,
 and `no` for disabling are no longer supported.
 
+==== Groovy dependencies
```

Comment:
Ha! I missed the total number of `=` signs. `====` instead of `===`. Sorry for the noise.